src/coordinates.rs

```diff
@@ -89,28 +89,24 @@ impl_from_integer!(u16);
 
 impl CoordinateNumber {
     pub fn gerber(&self, format: &CoordinateFormat) -> Result<String, GerberError> {
-        // Format invariants
-        if format.decimal > DECIMAL_PLACES_CHARS {
-            return Err(GerberError::CoordinateFormatError("Invalid precision: Too high!".into()))
+        fn rounded_div(dividend: i64, divisor: i64) -> i64 {
```

Comment:
Not sure if we should move this method to another location... It would be great to have this functionality in the Rust standard library :)